graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/TopSecondDegreeByCountRequest.java

```diff
@@ -77,6 +79,33 @@ public TopSecondDegreeByCountRequest(
     this.maxTweetSocialProofSize = maxTweetSocialProofSize;
     this.minUserSocialProofSizes = minUserSocialProofSizes;
     this.resultFilterChain = resultFilterChain;
+    this.socialProofTypeUnions = Collections.emptySet();
+  }
+
+  public TopSecondDegreeByCountRequest(
```

Comment:
could you add the new argument in the original request constructor? and clients can assign socialProofTypeUnions to emptySet if it is None in the thrift request. adding a new constructor for each new argument will be hard to maintain later on.